src/cmd/cluster_intel_collector/tarball.go

```diff
@@ -0,0 +1,81 @@
+package cluster_intel_collector
```

Comment:
there are no unit tests for this file.
if there are specific unit tests they should be in `tarball_test.go`

Reply:
The tarball is indirectly part of the log, objects and version collection tests. I'm aware that this way tests are not strictly limited to one unit, OTH unit tests for the tarball only would very much look the same as those tests, so it feels redundant.